ReactiveCocoaTests/InterceptingSpec.swift

```diff
@@ -668,19 +668,19 @@ class InterceptingSpec: QuickSpec {
 				}
 
 				func validate(arguments: [Any?], offset: UInt) {
-					expect(arguments[0] as? CChar) == CChar.max - CChar(offset)
-					expect(arguments[1] as? CShort) == CShort.max - CShort(offset)
-					expect(arguments[2] as? CInt) == CInt.max - CInt(offset)
-					expect(arguments[3] as? CLong) == CLong.max - CLong(offset)
-					expect(arguments[4] as? CLongLong) == CLongLong.max - CLongLong(offset)
-					expect(arguments[5] as? CUnsignedChar) == CUnsignedChar.max - CUnsignedChar(offset)
-					expect(arguments[6] as? CUnsignedShort) == CUnsignedShort.max - CUnsignedShort(offset)
-					expect(arguments[7] as? CUnsignedInt) == CUnsignedInt.max - CUnsignedInt(offset)
-					expect(arguments[8] as? CUnsignedLong) == CUnsignedLong.max - CUnsignedLong(offset)
-					expect(arguments[9] as? CUnsignedLongLong) == CUnsignedLongLong.max - CUnsignedLongLong(offset)
-					expect(arguments[10] as? CFloat) == CFloat.greatestFiniteMagnitude - CFloat(offset)
-					expect(arguments[11] as? CDouble) == CDouble.greatestFiniteMagnitude - CDouble(offset)
-					expect(arguments[12] as? CBool) == (offset % 2 == 0 ? true : false)
+					expect((arguments[0] as! NSNumber).int8Value) == CChar.max - CChar(offset)
```

Comment:
> NSNumber, NSValue and NSNull bridging is not a thing until Swift 3.0.1. 

Ooooh TIL.
I like your change then, since one day we'll be able to drop the `#if` 👍 Thanks.